src/UICircularProgressRing/UICircularProgressRing.swift

```diff
@@ -1099,7 +1125,14 @@ fileprivate extension CALayer {
 
         let currentTime = ringLayer.convertTime(CACurrentMediaTime(), from: nil)
         ringLayer.beginTime = currentTime - pauseTime
-
+        
+        //Create a new completion timer
+        completionTimer = Timer.scheduledTimer(timeInterval: currentTime - pauseTime,
```

Comment:
same thing here, use `scheduledTimer(withTimeInterval:repeats:block:)` instead to create the timer

Reply:
ignore, only supported in iOS 10.0